test/test_models/test_peft_sam.py

```diff
@@ -1,17 +1,32 @@
 import unittest
 
+from micro_sam.models.peft_sam import FacTSurgery, LoRASurgery
```

Comment:
I would move these imports in the corresponding tests. The idea here is to move the imports of the functionality being tested into the tests, so that we get a specific failure in the test if the import fails.

Reply:
Okay, yes. That's important. Thanks for the hint!